include/EASTL/sort.h

```diff
@@ -738,15 +813,39 @@ namespace eastl
 		// This is because the implementation below makes an assumption about the input
 		// data that quick_sort satisfies but arbitrary data may not.
 		// There is a standalone insertion_sort function. 
-		template <typename RandomAccessIterator, typename Compare>
+		template <typename RandomAccessIterator, typename Compare, typename eastl::enable_if<!eastl::is_lvalue_reference<typename eastl::iterator_traits<RandomAccessIterator>::value_type>::value>::type* = nullptr >
```

Comment:
Oh not a typo-- re-read your comment and what I wrote... I don't remember why I used lvalue/rvalue for this case but it looks wrong. I changed it is_move_constructible instead.